src/components/imageslider/imageslider.styles.css

```diff
@@ -0,0 +1,31 @@
+.slide {
+    height: 80vh;
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+    cursor: pointer;
+}
+
+.arrow {
+    cursor: pointer;
+    position: absolute;
+    top: 50%;
+    margin-top: -35px;
```

Comment:
You could use rems instead of pixels

Reply:
I have replaced rem unit in place of pixel unit.